src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -110,8 +106,7 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
       alertSetting: `${Actions.CUSTOMIZED_ALERTS}`,
       metric: MetricValues.ERRORS,
       interval: '',
-      placeholder: '',
-      threshold: '',
+      threshold: '10',
```

Comment:
mmm I would pull this out into a local const so it's more clear this is our default value we've chosen

Reply:
Done.